docs/app/Examples/views/Card/Content/CardExampleLinkCard.js

```diff
@@ -2,12 +2,20 @@ import React from 'react'
 import { Card } from 'semantic-ui-react'
 
 const CardExampleLinkCard = () => (
-  <Card
-    href='#link'
-    header='Elliot Baker'
-    meta='Friend'
-    description='Elliot is a sound engineer living in Nashville who enjoys playing guitar and hanging with his cat.'
-  />
+  <Card.Group>
+    <Card
+      href='#link'
+      header='Elliot Baker'
+      meta='Friend'
+      description='Elliot is a sound engineer living in Nashville who enjoys playing guitar and hanging with his cat.'
+    />
+    <Card
+      link
+      header='Rick Sanchez'
+      meta='Scientist'
+      description='Rick is a genius scientist whose alcoholism and reckless, nihilistic behavior are a source of concern for his family'
+    />
+  </Card.Group>
```

Comment:
You can make these appear together while still having separate example files.  The advantage is that each example will have its own code editor.

To do this, create a new file and add it to the `index.js` in this directory just after the `CardExampleLinkCard.js` entry, but do not include a `title` nor `description`:

**docs/app/Examples/views/Card/Content/index.js**
```jsx
    <ComponentExample
      title='Link'
      description='A card can be formatted to link to other content.'
      examplePath='views/Card/Content/CardExampleLinkCard'
    />
    <ComponentExample
      examplePath='views/Card/Content/CardExampleLinkCardProp'
    />
```

The second example will render directly below the title/description of the first example.  This is common in our docs for grouping examples.